app/controllers/coronavirus_local_restrictions_controller.rb

```diff
@@ -36,8 +51,18 @@ def results
                       breadcrumbs: breadcrumbs,
                     }
     elsif @location_lookup.invalid_postcode?
+      GovukError.notify(
+        "Postcode failed Mapit validation",
+        extra: { error_message: "Postcode #{@postcode} failed validation with Mapit" },
```

Comment:
Should we include the postcode to sentry? what if it includes PII?

Reply:
That's a tough one. We have the postcode in the Kibana logs when the checker is running, because we're logging every POST request. I suppose if we don't log it, how will we know what the actual problem is? Do we need to know?

Reply:
correct me if my understanding is wrong but should we only log to sentry when there is an actual error - invalid postcodes are handled appropriately. I think sentry should only get errors when there is an error with the API or another unknown error

Reply:
One of the questions in the trello card is:

>What we don't know
  We don't know what users are putting in the postcode look up box to trigger an error message
  We don't know whether this is a tech or content problem. Is there a problem with mapit? Is it out of date?

I was hoping that if we log a more detailed error, we could answer some of those questions.